app/models/identity_store.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'common/models/base'
+require 'common/models/redis_store'
+
+# Stores attributes used to identify a user. Serves as a set of inputs to an MVI lookup. Also serves
+# as the receiver of identity attributes received from alternative sources during the SSO flow.
+class IdentityStore < Common::RedisStore
```

Comment:
I think this model is perfect for the goals here. I was thinking though, this is extremely specific to:
- Sidekiq
- Basic User attributes

So I'd name it something that reflected what it's intended for. For example, doesn't have to be this, but something like `UserProfileAttributes`